examples/applications/plot_face_recognition.py

```diff
@@ -108,7 +108,7 @@
     "C": [1e3, 5e3, 1e4, 5e4, 1e5],
     "gamma": [0.0001, 0.0005, 0.001, 0.005, 0.01, 0.1],
 }
-clf = GridSearchCV(SVC(kernel="rbf", class_weight="balanced"), param_grid)
```

Comment:
Alternatively trying `RandomizedSearchCV` with `n_iter=10` should work well here and be faster than an exhaustive grid search of the 5x5 combinations.

Reply:
Thanks for this idea! Yeah, totally makes sense! I will try this option and report performance differences I register.